packages/flutter_tools/test/general.shard/asset_bundle_test.dart

```diff
@@ -36,6 +36,7 @@ void main() {
     file.writeAsStringSync(
       json.encode(<String, dynamic>{
         'packages': <dynamic>[
+          <String, dynamic>{'name': 'my_app', 'rootUri': '../', 'packageUri': 'lib/'},
           ...packages.entries.map((MapEntry<String, String> entry) {
             return <String, dynamic>{
               'name': entry.key,
```

Comment:
This is a good example, there are so many copy-pastes of this function across the flutter_tools tests.